code/game/machinery/airlock_cycle_control.dm

```diff
@@ -76,7 +76,7 @@
 	var/scan_on_late_init = FALSE
 	var/depressurization_margin = 10 // use a lower value to reduce cross-contamination
 	var/overlays_hash = null
-	var/skip_delay = 150
```

Comment:
This may be changed due to the change first being applied on NSV13's rebase code, which could be up to a week or two out of date with Bee. Will revert on request.

Reply:
please do

Reply:
Yeah please keep it at 150. 30 seconds is way too long.

Reply:
It has been reverted!